### PR TITLE
fix: Provide working Dockerfile

### PR DESCRIPTION
**Pull request by tomkerkhove**
Co-authored-by: Tom Kerkhove <kerkhove.tom@gmail.com>Signed-off-by: Tom Kerkhove <kerkhove.tom@gmail.com>

### CODE REVIEW

File: src/Arcus.Messaging.Tests.Runtimes.AzureFunction.ServiceBus.Queue/MessageHandlers/DummyServiceBusMessageHandler.cs [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+using Arcus.Messaging.Abstractions;
+using Arcus.Messaging.Abstractions.ServiceBus;
+using Arcus.Messaging.Abstractions.ServiceBus.MessageHandling;
+using Arcus.Messaging.Tests.Core.Messages.v1;
+using GuardNet;
+using Microsoft.Extensions.Logging;
+
+namespace Arcus.Messaging.Tests.Runtimes.AzureFunction.ServiceBus.Queue.MessageHandlers
+{
+    public class DummyServiceBusMessageHandler : IAzureServiceBusMessageHandler<Order>
```
Comment (tomkerkhove) [inline src/Arcus.Messaging.Tests.Runtimes.AzureFunction.ServiceBus.Queue/MessageHandlers/DummyServiceBusMessageHandler.cs:12]: This is temporary and will be removed

File: src/Arcus.Messaging.Tests.Runtimes.AzureFunction.ServiceBus.Queue/Startup.cs [hunk 2]
```diff
@@ -18,26 +18,26 @@ public class Startup : FunctionsStartup
         /// <param name="builder">The instance to build the registered services inside the functions app.</param>
         public override void Configure(IFunctionsHostBuilder builder)
         {
```
Comment (tomkerkhove) [inline src/Arcus.Messaging.Tests.Runtimes.AzureFunction.ServiceBus.Queue/Startup.cs:20]: The issue lies here at startup, somehow.

File: src/Arcus.Messaging.Tests.Runtimes.AzureFunction.ServiceBus.Queue/Startup.cs [hunk 3]
```diff
@@ -18,13 +19,12 @@ public class Startup : FunctionsStartup
...
-            IConfiguration configuration =
-                new ConfigurationBuilder()
+            IConfiguration configuration = new ConfigurationBuilder()
                     .SetBasePath(builder.GetContext().ApplicationRootPath)
                     .AddEnvironmentVariables("ARCUS_")
-                    .AddJsonFile("local.settings.json")
+                    .AddJsonFile("local.settings.json", optional: false)
```
Comment (tomkerkhove) [inline src/Arcus.Messaging.Tests.Runtimes.AzureFunction.ServiceBus.Queue/Startup.cs:24]: Time wasted : 9000

Comment (stijnmoreels) [reply src/Arcus.Messaging.Tests.Runtimes.AzureFunction.ServiceBus.Queue/Startup.cs]: Uh? That's so weird. Maybe also the namespace of the startup file.

Comment (tomkerkhove) [reply src/Arcus.Messaging.Tests.Runtimes.AzureFunction.ServiceBus.Queue/Startup.cs]: The file is not there so it blew up, but didn't give an error..

**Review by stijnmoreels — APPROVED** [review]

**Pull request merged**